### PR TITLE
40 social sign in sign up

### PR DESCRIPTION
**Pull request by evrusha**

### CODE REVIEW

File: .github/workflows/trello.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+name: GitHub Commit To Trello Comment
+
+on: [push, pull_request]
```
Comment (fooooxmr) [inline .github/workflows/trello.yml:3]: I think, that this file is not related to the current PR. Why it here?

File: app/controllers/users/confirmations_controller.rb [hunk 2]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::ConfirmationsController < Devise::ConfirmationsController
```
Comment (fooooxmr) [inline app/controllers/users/confirmations_controller.rb:3]: Why do we need a class, which we are not overriding?

File: app/controllers/users/passwords_controller.rb [hunk 3]
```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+class Users::PasswordsController < Devise::PasswordsController
```
Comment (fooooxmr) [inline app/controllers/users/passwords_controller.rb:3]: The same. Looks like we do not need this class.

File: app/controllers/users/registrations_controller.rb [hunk 4]
```diff
@@ -0,0 +1,62 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
```
Comment (fooooxmr) [inline app/controllers/users/registrations_controller.rb:3]: The same as before.

File: app/controllers/users/unlocks_controller.rb [hunk 5]
```diff
@@ -0,0 +1,30 @@
+# frozen_string_literal: true
+
+class Users::UnlocksController < Devise::UnlocksController
```
Comment (fooooxmr) [inline app/controllers/users/unlocks_controller.rb:3]: The same.

File: app/views/devise/shared/_links.html.slim [hunk 6]
```diff
@@ -19,6 +19,8 @@
...
 
 - if devise_mapping.omniauthable?
-  - resource_class.omniauth_providers.each do |provider|
-    = link_to "#{t('data.with')} #{OmniAuth::Utils.camelize(provider)}", omniauth_authorize_path(resource_name, provider), method: :post
-    br
+  = form_for 'Login', url: user_google_oauth2_omniauth_authorize_path, method: :post do |f|
+    div = f.submit 'Login', type: :image, src: url_for('/assets/btn_google_signin_dark_pressed_web.png')
+  = form_for 'Login', url: user_facebook_omniauth_authorize_path, method: :post do |f|
+    div = f.submit 'Login', type: :image, src: url_for('/assets/1485482214-facebook_78681.png')
+
```
Comment (fooooxmr) [inline app/views/devise/shared/_links.html.slim:26]: Empty line

**Review by fooooxmr — APPROVED** [review]

File: .github/workflows/trello.yml (hunk 1 above)
Comment (evrusha) [reply .github/workflows/trello.yml]: I chose the wrong base to merge. Fixed

File: app/controllers/users/confirmations_controller.rb (hunk 2 above)
Comment (evrusha) [reply app/controllers/users/confirmations_controller.rb]: Fixed

File: app/controllers/users/passwords_controller.rb (hunk 3 above)
Comment (evrusha) [reply app/controllers/users/passwords_controller.rb]: Fixed

File: app/controllers/users/registrations_controller.rb (hunk 4 above)
Comment (evrusha) [reply app/controllers/users/registrations_controller.rb]: Fixed

File: app/controllers/users/unlocks_controller.rb (hunk 5 above)
Comment (evrusha) [reply app/controllers/users/unlocks_controller.rb]: Fixed

File: app/views/devise/shared/_links.html.slim (hunk 6 above)
Comment (evrusha) [reply app/views/devise/shared/_links.html.slim]: Fixed

**Review by ritamma — APPROVED** [review]

**Review by Terapool — APPROVED** [review]

**Review by Dmitryshl — APPROVED** [review]
